admin_guide/custom_resource_definitions.adoc

```diff
@@ -23,14 +23,24 @@ Deploying a CRD into the cluster causes the Kubernetes API server to begin servi
 
 When you create a new custom resource definition (CRD), the Kubernetes API Server
 reacts by creating a new RESTful resource path, that can be accessed by an entire cluster
-or a single project (namespace). As with existing built-in objects, deleting a
-project deletes all custom objects in that project.
+or a single project (namespace). If the CRD can be accessed by an entire cluster
+and you use certain standard cluster roles, admin, edit, and view, you can use 
```

Comment:
Wording here is a bit weird.

If you want to grant access to the CRD to normal users, the simplest way is to leverage cluster role aggregation via the standard cluster roles admin, edit, and view.  admin or edit is generally assigned to the user that created a project via the project template.  Cluster role aggregation allows the insertion of custom policy rules into these cluster roles.  This seamlessly integrates the new resource into the cluster's RBAC policy as if it was a built-in resource.

Reply:
Thanks! I think the entire change should be replaced with this text:

or a single project (namespace). As with existing built-in objects, deleting a project deletes all custom objects in that project.

If you want to grant access to the CRD to users, use cluster role aggregation to grant access to users with the admin, edit, or view default cluster roles. Cluster role aggregation allows the insertion of custom policy rules into these cluster roles. This behavior seamlessly integrates the new resource into the cluster's RBAC policy as if it was a built-in resource.